extensions/amp-carousel/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "paths": {}
+  },
+  "include": ["0.1/*.js", "0.1/*.d.ts", "../../src/*.d.ts"],
```

Comment:
Let's move this file into `0.1` and make each of these `**/*.js` and `**/*.d.ts`, that way the implication is "if a directory has a `tsconfig.js`, it is type-checked" (except root)

Reply:
sure